src/test/resources/expected-classes/scala/deprecated/Status.scala.txt

```diff
@@ -1,5 +1,7 @@
 package com.github.graphql
 
+import com.fasterxml.jackson.core.`type`.TypeReference
```

Comment:
Do you also need to do the “if JACKSON” check?

Reply:
Only enumeration is needed, and only when Jackson is used